model/generate.go

```diff
@@ -158,13 +158,24 @@ func (g *GeneratedProject) NewVersion() (*Project, *ParserProject, *Version, *ta
 }
 
 func (g *GeneratedProject) Save(ctx context.Context, p *Project, pp *ParserProject, v *Version, t *task.Task, pm *projectMaps) error {
+
+	//if pp.ConfigUpdateNumber > v.ConfigUpdateNumber { // I don't think this could happen until UseParserProject, and even then maybe not until we stop preferencing version
```

Comment:
If we hit the else block, then `pp.ConfigUpdateNumber == v.ConfigUpdateNumber`. When `UseParserProject == true`, does that somehow cause `v.ConfigUpdateNumber` no longer to be incremented, which (eventually) causes `pp.ConfigUpdateNumber > v.ConfigUpdateNumber`?

Also, for new projects, will `pp.ConfigUpdateNumber` already be > 0, which will cause us to hit this branch?